### ISSUE TITLE
make check should also test tools

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1757]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1753
- Created at 2013-01-10 17:46:03 by [simo](https://pagure.io/user/simo)
- Closed at 2020-03-24 14:12:13 as wontfix
- Assigned to [preichl](https://pagure.io/user/preichl)
---

Tools like sss_cache are relatively easy to test, make check should have additional tests that set up data in the ldb cache and then run our tools forking them off, finally check the data in teh cache reflect the outcome we expect.

### Comments

---

**Comment from [simo](https://pagure.io/user/simo) at [2013-01-17 15:16:27](https://pagure.io/SSSD/sssd/issue/1753#243177)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.10 beta
priority: major => minor
rhbz:  => 0
selected:  => Want
type: defect => task

---

**Comment from [simo](https://pagure.io/user/simo) at [2013-03-28 14:52:53](https://pagure.io/SSSD/sssd/issue/1753#243178)**

Not critical for beta and only affects unit tests. Can be done in 1.10 proper.

milestone: SSSD 1.10 beta => SSSD 1.10.0
review:  => 0

---

**Comment from [simo](https://pagure.io/user/simo) at [2013-06-13 16:48:36](https://pagure.io/SSSD/sssd/issue/1753#243179)**

Fields changed

milestone: SSSD 1.10.0 => SSSD 1.10.1

---

**Comment from [simo](https://pagure.io/user/simo) at [2013-07-16 20:15:30](https://pagure.io/SSSD/sssd/issue/1753#243180)**

I doubt this is going to make 1.10.1. I would prefer to introduce tests for the tools along with improving the test coverage during the F-20 cycle.

changelog:  => 
milestone: SSSD 1.10.1 => NEEDS_TRIAGE

---

**Comment from [simo](https://pagure.io/user/simo) at [2013-07-18 15:31:44](https://pagure.io/SSSD/sssd/issue/1753#243181)**

Fields changed

milestone: NEEDS_TRIAGE => Interim Bucket

---

**Comment from [simo](https://pagure.io/user/simo) at [2013-07-30 12:54:43](https://pagure.io/SSSD/sssd/issue/1753#243182)**

Fields changed

milestone: Interim Bucket => SSSD 1.12 beta

---

**Comment from [simo](https://pagure.io/user/simo) at [2014-02-19 11:52:24](https://pagure.io/SSSD/sssd/issue/1753#243183)**

Fields changed

blocking:  => #1923

---

**Comment from [simo](https://pagure.io/user/simo) at [2014-05-28 13:23:46](https://pagure.io/SSSD/sssd/issue/1753#243184)**

Fields changed

review: 0 => 1

---

**Comment from [simo](https://pagure.io/user/simo) at [2014-05-30 15:59:09](https://pagure.io/SSSD/sssd/issue/1753#243185)**

Fields changed

milestone: SSSD 1.12 beta => SSSD 1.13 beta

---

**Comment from [simo](https://pagure.io/user/simo) at [2014-09-26 18:49:57](https://pagure.io/SSSD/sssd/issue/1753#243186)**

Fields changed

mark:  => 0

---

**Comment from [simo](https://pagure.io/user/simo) at [2015-01-06 17:56:22](https://pagure.io/SSSD/sssd/issue/1753#243187)**

Fields changed

owner: somebody => preichl

---

**Comment from [simo](https://pagure.io/user/simo) at [2015-02-10 17:25:55](https://pagure.io/SSSD/sssd/issue/1753#243188)**

Fields changed

milestone: SSSD 1.13 beta => SSSD 1.13 backlog

---

**Comment from [simo](https://pagure.io/user/simo) at [2015-02-12 20:26:49](https://pagure.io/SSSD/sssd/issue/1753#243189)**

Mass-moving tickets not planned for the next two releases.

Please reply with a comment if you disagree about the move..

milestone: SSSD 1.13 backlog => SSSD 1.15 beta

---

**Comment from [simo](https://pagure.io/user/simo) at [2017-02-24 15:08:15](https://pagure.io/SSSD/sssd/issue/1753#243190)**

**Metadata Update from @simo**:
- Issue assigned to preichl
- Issue marked as blocked by: #1923
- Issue set to the milestone: SSSD Future releases (no date set yet)

---

**Comment from [simo](https://pagure.io/user/simo) at [2020-03-13 14:16:16](https://pagure.io/SSSD/sssd/issue/1753#633177)**

**Metadata Update from @thalman**:
- Custom field blocking reset (from #1923)
- Custom field design_review reset (from 0)
- Custom field mark reset (from 0)
- Custom field patch reset (from 0)
- Custom field review adjusted to on (was: 1)
- Custom field testsupdated reset (from 0)
- Issue close_status updated to: None
- Issue tagged with: Canditate to close

---

**Comment from [simo](https://pagure.io/user/simo) at [2020-03-24 14:12:12](https://pagure.io/SSSD/sssd/issue/1753#636682)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [simo](https://pagure.io/user/simo) at [2020-03-24 14:12:13](https://pagure.io/SSSD/sssd/issue/1753#636683)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
